server/game/cards/events/01/unbowedunbentunbroken.js

```diff
@@ -41,13 +38,14 @@ class UnbowedUnbentUnbroken extends DrawCard {
             effect: ability.effects.cannotInitiateChallengeType(challengeType)
         }));
 
-        this.game.addMessage('{0} uses {1} to make {2} unable to initiate {3} challenges until the end of the phase', player, this, otherPlayer, challengeType);
+        this.game.addMessage('{0} plays {1} to make {2} unable to initiate {3} challenges until the end of the phase', player, this, otherPlayer, challengeType);
 
         return true;
     }
 
     cancel(player) {
-        this.game.addMessage('{0} cancel the resolution of {1}', player, this);
+        //I really want to get rid of stuff like this
```

Comment:
I know your stance on this, I just have a lot of issues with it. :) It's only a handful cards in the entire cardpool that are implemented like this, and adding such functionality across the board would add extra clicks, cause problematic interactions (do you want to give new players/grievers the ability to cancel Pentoshi's When Revealed?) and be a shitload of work to boot.

Fair enough about the comment. I'll record it somewhere else. :)